### PR TITLE
Remove leftover TODO from stored procs guide

### PR DESCRIPTION
**Pull request by lgreene27**
## Description & motivation
Removing a leftover TODO from the stored procs guide to add links to the star macro documentation. This is a tiny tiny change.

### CODE REVIEW

File: website/docs/guides/migration/tools/migrating-from-stored-procedures/3-mapping-updates.md [hunk 1]
```diff
@@ -52,4 +52,4 @@ SELECT
 FROM {{ ref(‘stg_orders’) }}
 ```
 
-The `dbt_utils.star()` macro (TODO add links) will print out the full list of columns in the table, but skip the ones I’ve listed in the except list, which allows me to perform the same logic while writing fewer lines of code. This is a simple example of using dbt macros to simplify and shorten your code, and dbt can get a lot more sophisticated as you learn more techniques.
+The `dbt_utils.star()` macro will print out the full list of columns in the table, but skip the ones I’ve listed in the except list, which allows me to perform the same logic while writing fewer lines of code. (Read more about the [dbt_utils package](https://hub.getdbt.com/dbt-labs/dbt_utils/latest/) and [star macro](https://github.com/dbt-labs/dbt-utils/tree/0.8.6/#star-source).) This is a simple example of using dbt macros to simplify and shorten your code, and dbt can get a lot more sophisticated as you learn more techniques.
```
Comment (KiraFuruichi) [inline website/docs/guides/migration/tools/migrating-from-stored-procedures/3-mapping-updates.md]:
```suggestion
The `dbt_utils.star()` macro will print out the full list of columns in the table, but skip the ones I’ve listed in the except list, which allows me to perform the same logic while writing fewer lines of code. This is a simple example of using dbt macros to simplify and shorten your code, and dbt can get a lot more sophisticated as you learn more techniques. Read more about the [dbt_utils package](https://hub.getdbt.com/dbt-labs/dbt_utils/latest/) and the [star macro here](https://github.com/dbt-labs/dbt-utils/tree/0.8.6/#star-source).
```

Just reordering the sentences to flow a little better :)

**Comment by KiraFuruichi** [comment]
Hey @lgreene27 thanks for putting this PR together! I just happened to notice while reviewing this PR that all the values that are supposed to be strings in the code snippets in this guide are not-quite strings (see screenshot below). This usually happens if someone ports over text from Google docs or something. 

Would you mind making all the string values in the code snippets in this stored proc guide actual single quotes (') vs what they are now? They happen in the Inserts, Updates, Merges, and Deletes pages of this guide. Totally cool to say no haha.

image

**Comment by lgreene27** [comment]
@KiraFuruichi I think I got all of the single quotes, but let me know if I missed any! Or if you see any other changes to make in the guide while I'm here

File: website/docs/guides/migration/tools/migrating-from-stored-procedures/2-mapping-inserts.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ To really break it down, let’s consider a simple example:
 ```sql
 INSERT INTO returned_orders (order_id, order_date, total_return)
 
-SELECT order_id, order_date, total FROM orders WHERE type = ‘return’
+SELECT order_id, order_date, total FROM orders WHERE type = `return`
```
Comment (KiraFuruichi) [inline website/docs/guides/migration/tools/migrating-from-stored-procedures/2-mapping-inserts.md]:
```suggestion
SELECT order_id, order_date, total FROM orders WHERE type = 'return'
```

File: website/docs/guides/migration/tools/migrating-from-stored-procedures/2-mapping-inserts.md [hunk 3]
```diff
@@ -21,9 +21,9 @@ SELECT
     order_date as order_date,
     total as total_return
 
-FROM {{ ref(‘orders’) }}
+FROM {{ ref(`orders`) }}
```
Comment (KiraFuruichi) [inline website/docs/guides/migration/tools/migrating-from-stored-procedures/2-mapping-inserts.md]:
```suggestion
FROM {{ ref('orders') }}
```

File: website/docs/guides/migration/tools/migrating-from-stored-procedures/2-mapping-inserts.md [hunk 4]
```diff
@@ -21,9 +21,9 @@ SELECT
     order_date as order_date,
     total as total_return
 
-FROM {{ ref(‘orders’) }}
+FROM {{ ref('orders') }}
 
-WHERE type = ‘return’
+WHERE type = `return`
```
Comment (KiraFuruichi) [inline website/docs/guides/migration/tools/migrating-from-stored-procedures/2-mapping-inserts.md]:
```suggestion
WHERE type = 'return'
```

File: website/docs/guides/migration/tools/migrating-from-stored-procedures/2-mapping-inserts.md [hunk 5]
```diff
@@ -47,11 +47,11 @@ INSERT INTO all_customers SELECT * FROM eu_customers
 The dbt-ified version of this would end up looking something like:
 
 ```sql
-SELECT * FROM {{ ref(‘us_customers’) }}
+SELECT * FROM {{ ref(`us_customers`) }}
```
Comment (KiraFuruichi) [inline website/docs/guides/migration/tools/migrating-from-stored-procedures/2-mapping-inserts.md]:
```suggestion
SELECT * FROM {{ ref('us_customers') }}
```

File: website/docs/guides/migration/tools/migrating-from-stored-procedures/2-mapping-inserts.md [hunk 6]
```diff
@@ -47,11 +47,11 @@ INSERT INTO all_customers SELECT * FROM eu_customers
...
 
 UNION ALL
 
-SELECT * FROM {{ ref(‘eu_customers’) }}
+SELECT * FROM {{ ref(`eu_customers`) }}
```
Comment (KiraFuruichi) [inline website/docs/guides/migration/tools/migrating-from-stored-procedures/2-mapping-inserts.md]:
```suggestion
SELECT * FROM {{ ref('eu_customers') }}
```

File: website/docs/guides/migration/tools/migrating-from-stored-procedures/3-mapping-updates.md [hunk 7]
```diff
@@ -8,7 +8,7 @@ id: 3-updates
 ```sql
 UPDATE orders
 
-SET type = ‘return’
+SET type = `return`
```
Comment (KiraFuruichi) [inline website/docs/guides/migration/tools/migrating-from-stored-procedures/3-mapping-updates.md]:
```suggestion
SET type = 'return'
```

File: website/docs/guides/migration/tools/migrating-from-stored-procedures/3-mapping-updates.md [hunk 8]
```diff
@@ -25,14 +25,14 @@ If I were building an intermediate transformation from the above query would tra
 ```sql
 SELECT
     CASE
-        WHEN total < 0 THEN ‘return’
+        WHEN total < 0 THEN `return`
```
Comment (KiraFuruichi) [inline website/docs/guides/migration/tools/migrating-from-stored-procedures/3-mapping-updates.md]:
```suggestion
        WHEN total < 0 THEN 'return'
```

File: website/docs/guides/migration/tools/migrating-from-stored-procedures/3-mapping-updates.md [hunk 9]
```diff
@@ -25,14 +25,14 @@ If I were building an intermediate transformation from the above query would tra
...
         ELSE type
     END AS type,
-    
+
     order_id,
     order_date
 
-FROM {{ ref(‘stg_orders’) }}
+FROM {{ ref(`stg_orders`) }}
```
Comment (KiraFuruichi) [inline website/docs/guides/migration/tools/migrating-from-stored-procedures/3-mapping-updates.md]:
```suggestion
FROM {{ ref('stg_orders') }}
```

File: website/docs/guides/migration/tools/migrating-from-stored-procedures/3-mapping-updates.md [hunk 10]
```diff
@@ -43,13 +43,13 @@ Another way I could have written the model a bit more dynamically might be:
 
 ```sql
 SELECT
-    {{ dbt_utils.star(from=ref(‘stg_orders’), except=[‘type’]) }},
+    {{ dbt_utils.star(from=ref(`stg_orders`), except=[`type`]) }},
```
Comment (KiraFuruichi) [inline website/docs/guides/migration/tools/migrating-from-stored-procedures/3-mapping-updates.md]:
```suggestion
    {{ dbt_utils.star(from=ref('stg_orders'), except=['type']) }},
```

File: website/docs/guides/migration/tools/migrating-from-stored-procedures/3-mapping-updates.md [hunk 11]
```diff
@@ -43,13 +43,13 @@ Another way I could have written the model a bit more dynamically might be:
...
     CASE
-        WHEN total < 0 THEN ‘return’
+        WHEN total < 0 THEN `return`
```
Comment (KiraFuruichi) [inline website/docs/guides/migration/tools/migrating-from-stored-procedures/3-mapping-updates.md]:
```suggestion
        WHEN total < 0 THEN 'return'
```

File: website/docs/guides/migration/tools/migrating-from-stored-procedures/3-mapping-updates.md [hunk 12]
```diff
@@ -43,13 +43,13 @@ Another way I could have written the model a bit more dynamically might be:
...
         ELSE type
     END AS type,
 
-FROM {{ ref(‘stg_orders’) }}
+FROM {{ ref(`stg_orders`) }}
```
Comment (KiraFuruichi) [inline website/docs/guides/migration/tools/migrating-from-stored-procedures/3-mapping-updates.md]:
```suggestion
FROM {{ ref('stg_orders') }}
```

File: website/docs/guides/migration/tools/migrating-from-stored-procedures/4-mapping-deletes.md [hunk 13]
```diff
@@ -14,15 +14,15 @@ DELETE FROM stg_orders WHERE order_status IS NULL
 In a dbt model, you’ll need to first identify the records that should be deleted and then filter them out. There are really two primary ways you might translate this query:
 
 ```sql
-SELECT * FROM {{ ref(‘stg_orders’) }} WHERE order_status IS NOT NULL
+SELECT * FROM {{ ref(`stg_orders`) }} WHERE order_status IS NOT NULL
```
Comment (KiraFuruichi) [inline website/docs/guides/migration/tools/migrating-from-stored-procedures/4-mapping-deletes.md]:
```suggestion
SELECT * FROM {{ ref('stg_orders') }} WHERE order_status IS NOT NULL
```

File: website/docs/guides/migration/tools/migrating-from-stored-procedures/4-mapping-deletes.md [hunk 14]
```diff
@@ -33,7 +33,7 @@ soft_deletes AS (
             ELSE false
         END AS to_delete
 
-    FROM {{ ref(‘stg_orders’) }}
+    FROM {{ ref(`stg_orders`) }}
```
Comment (KiraFuruichi) [inline website/docs/guides/migration/tools/migrating-from-stored-procedures/4-mapping-deletes.md]:
```suggestion
    FROM {{ ref('stg_orders') }}
```

File: website/docs/guides/migration/tools/migrating-from-stored-procedures/5-mapping-merges.md [hunk 15]
```diff
@@ -49,7 +49,7 @@ using_clause AS (
         subtotal,
         tip
 
-    FROM {{ ref(‘rides_to_load’) }}
+    FROM {{ ref(`rides_to_load`) }}
```
Comment (KiraFuruichi) [inline website/docs/guides/migration/tools/migrating-from-stored-procedures/5-mapping-merges.md]:
```suggestion
    FROM {{ ref('rides_to_load') }}
```

File: website/docs/guides/migration/tools/migrating-from-stored-procedures/5-mapping-merges.md [hunk 16]
```diff
@@ -97,16 +97,16 @@ We can add the following `config()` block to the top of our model to specify how
 ```sql
 {{
     config(
-        materialized=‘incremental’,
-        unique_key=‘ride_id’,
-        incremental_strategy=‘merge’
+        materialized=`incremental`,
```
Comment (KiraFuruichi) [inline website/docs/guides/migration/tools/migrating-from-stored-procedures/5-mapping-merges.md]:
```suggestion
        materialized='incremental',
```

File: website/docs/guides/migration/tools/migrating-from-stored-procedures/5-mapping-merges.md [hunk 17]
```diff
@@ -97,16 +97,16 @@ We can add the following `config()` block to the top of our model to specify how
...
+        unique_key=`ride_id`,
```
Comment (KiraFuruichi) [inline website/docs/guides/migration/tools/migrating-from-stored-procedures/5-mapping-merges.md]:
```suggestion
        unique_key='ride_id',
```

File: website/docs/guides/migration/tools/migrating-from-stored-procedures/5-mapping-merges.md [hunk 18]
```diff
@@ -97,16 +97,16 @@ We can add the following `config()` block to the top of our model to specify how
...
+        incremental_strategy=`merge`
```
Comment (KiraFuruichi) [inline website/docs/guides/migration/tools/migrating-from-stored-procedures/5-mapping-merges.md]:
```suggestion
        incremental_strategy='merge'
```

File: website/docs/guides/migration/tools/migrating-from-stored-procedures/5-mapping-merges.md [hunk 19]
```diff
@@ -97,16 +97,16 @@ We can add the following `config()` block to the top of our model to specify how
...
     )
 }}
 ```
 
 The three configuration fields in this example are the most important ones.
 
-- Setting `materialized=’incremental’` tells dbt to apply UPSERT logic to the target table.
+- Setting `materialized=`incremental`` tells dbt to apply UPSERT logic to the target table.
```
Comment (KiraFuruichi) [inline website/docs/guides/migration/tools/migrating-from-stored-procedures/5-mapping-merges.md]:
```suggestion
- Setting `materialized='incremental'` tells dbt to apply UPSERT logic to the target table.
```

File: website/docs/guides/migration/tools/migrating-from-stored-procedures/5-mapping-merges.md [hunk 20]
```diff
@@ -131,9 +131,9 @@ using_clause AS (
         subtotal,
         tip,
         max(load_timestamp) as load_timestamp
-    
-    FROM {{ ref(‘rides_to_load’) }}
- 
+
+    FROM {{ ref(`rides_to_load`) }}
```
Comment (KiraFuruichi) [inline website/docs/guides/migration/tools/migrating-from-stored-procedures/5-mapping-merges.md]:
```suggestion
    FROM {{ ref('rides_to_load') }}
```

**Review by KiraFuruichi — APPROVED** [review]
This looks great! Thank you for opening up this PR!!

**Pull request merged**